apps/knowledge-base/src/templates/layout.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Logo from '~/components/logo';
+
+const Container = styled.section`
+  height: auto;
+  min-height: 65vh;
+  max-width: 768px;
+  padding: 2rem 3rem;
+  margin: 30px auto;
+
+  background-color: #fff;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 0px 20px;
+
+  &:before {
+    content: '';
+    position: fixed;
```

Comment:
Optional: Other marketplace apps have backgrounds that scroll, I think it would be nice to have consistent behavior. But totally up to you!